<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pluginSettingDialog</class>
 <widget class="QDialog" name="pluginSettingDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统监控插件设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,5,1,1">
   <item>
    <widget class="QGroupBox" name="displayContentSettingGroupBox">
     <property name="title">
      <string>高效模式显示内容设置</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>41</y>
        <width>147</width>
        <height>86</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="onlyNetSpeedRadioButton">
         <property name="text">
          <string>只显示网速</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="onlyCPUMEMRadioButton">
         <property name="text">
          <string>只显示CPU、内存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="showAllRadioButton">
         <property name="text">
          <string>显示CP&amp;U、内存和网速</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fashionDisplayContentSettingGroupBox">
     <property name="title">
      <string>时尚模式显示内容设置</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>57</y>
        <width>123</width>
        <height>56</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="fashionOnlyCPUMEMRadioButton">
         <property name="text">
          <string>只显示CPU、内存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="fashionOnlyNetSpeedRadioButton">
         <property name="text">
          <string>只显示网速</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>行高：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="lineHeightSpinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>fashionDisplayContentSettingGroupBox</zorder>
  <zorder>displayContentSettingGroupBox</zorder>
  <zorder>buttonBox</zorder>
  <zorder>lineHeightSpinBox</zorder>
  <zorder>label</zorder>
  <zorder>horizontalSpacer</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>pluginSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>pluginSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
